pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -212,9 +212,11 @@ func ApplyFeatureGates() {
 
 	// Prioritizes nodes that satisfy pod's resource limits
 	if utilfeature.DefaultFeatureGate.Enabled(features.ResourceLimitsPriorityFunction) {
+		glog.Warningf("Registering resourcelimits priority function")
```

Comment:
Why is this a warning instead of an info log?

Reply:
It was copy-paste error as I copied this from earlier block. I will change it. Thanks.

Reply:
Done.